docs/pages/project/contributing/contributing-docs.md

```diff
@@ -24,15 +24,101 @@ Meshery documentation is made of these components:
 
 ## Set up your development environment
 
-{% include alert.html type="info" title="Jekyll" content="The Meshery Docs site is built using Jekyll - a simple static site generator. You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)." %}
+{% include alert.html type="info" title="Jekyll" content="The Meshery Docs site is built using Jekyll - a simple static site generator. Jekyll can be installed in the different platforms like Windows, Linux, and MacOs by the following steps:" %}
 
-- First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler.
+### For Windows
 
-**Note:** Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also installing Ruby Version Manager [RVM](https://rvm.io). RVM is a command-line tool which allows you to work with multiple Ruby environments on your local machine. Alternatively, if you're running Windows 10 version 1903 Build 18362 or higher, you can upgrade to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and run Jekyll in Linux instead.
```

Comment:
Ideally, we're not repeating any instructions that the user can get from jekyllrb.com. Let's aim for only listing instructions here that are unique to our project. Otherwise, let's point contributors to the docs for those dependencies.